Assets/Scripts/Game/Entities/PlayerEntity.cs

```diff
@@ -432,8 +444,8 @@ public bool IntermittentEnemySpawn(uint Minutes)
 
         public void SpawnCityGuards(bool forceSpawn)
         {
-            // Don't spawn if more than 10 enemies are already in the area
-            if (UnityEngine.Object.FindObjectsOfType<DaggerfallEntityBehaviour>().Length > 10)
+            // Don't spawn if more than 10 guards are already in the area
+            if (GameManager.Instance.HowManyEnemiesOfType(MobileTypes.Knight_CityWatch) > 10)
```

Comment:
This is a good fix. I've been looking into adding a serializable entity type to mobile NPCs as well so they can be saved/loaded and affected by spells.